package.json

```diff
@@ -36,7 +36,6 @@
     "release:publish": "npm run util:push-tags && npm publish && ts-node ./support/releaseToGitHub.ts",
     "start": "concurrently --kill-others --raw \"tsc --project ./tsconfig-demos.json --watch\" \"npm:build --dev --watch --serve\"",
     "test": "npm run util:run-tests && npm run test:prerender",
-    "test2": "npm run util:copy-icons && stencil test --no-docs --spec --e2e calcite-tooltip",
```

Comment:
@jcfranco do you know if we could have a script to test an individual test file?

Reply:
I know “npm run-script test calcite-whatever” _used to _work__ but stopped at some point.

Reply:
Yeah it doesn't seem to accept the argument since we modified the script to call another script. Maybe we can pass an  argument another way? @jcfranco

Reply:
I'll refactor the script. In the meantime you can use `npm run test:watch -- <calcite-whatevs>`.